packages/core/src/index.html

```diff
@@ -269,6 +269,7 @@
                 "
             ></footer>
         </div>
+        <div id="wallets_modal_responsive_root" class="wallets-modal-responsive-root"></div>
```

Comment:
Use the default `wallets_modal_root` instead then 😅 , the problem was if we have the root in html then clicking header will hide the modal cc: @adrienne-deriv

Reply:
@farhan-nurzi-deriv I can't :) If we could do this we don't need to create wallets_modal_responsive_root at all. in this case I'll need to change ModalProvider or add custom container for portal. This container was created exactly for responsive